README.md

```diff
@@ -1,8 +1,21 @@
 # Knot.x Fragments Chrome Extension
-Extends the Developer Tools, adding a sidebar that displays Fragments data associated with the
-selected DOM element.
+Extends the Developer Tools, adding a sidebar that displays [Fragments](https://github.com/Knotx/knotx-fragments) 
+data associated with the selected DOM element.
 
-// TODO why do we need this?
+<p align="center">
+  <img src="assets/images/preview.gif" alt="Knot.x Fragments Chrome Extension"/>
+</p>
+
+It is a bridge between the problem space (domain) and the space of solutions. Domain experts can 
```

Comment:
Is this some kind of business terminology? I, for instance, don't really know what it means. If I were to read this not knowing what our tool is then this specialist-sounding phrase could scare me out